src/pages/workspaces/workspace/Data.js

```diff
@@ -926,6 +926,7 @@ const WorkspaceData = _.flow(
 
   refresh() {
     this.setState(({ refreshKey }) => ({ refreshKey: refreshKey + 1 }))
+    this.loadMetadata()
```

Comment:
The `onSuccess` callback of the `EntityUploader` calls both `this.refresh` and `this.loadMetadata`, so we should remove the extra call from there.